pallets/allocations/Cargo.toml

```diff
@@ -35,8 +38,10 @@ pallet-balances = { git = "https://github.com/paritytech/substrate", default-fea
 sp-io = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
 sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
 sp-std = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
+pallet-membership = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.20" }
```

Comment:
I suggest moving this to dev-dependencies

Reply:
Done